code/src/components/Description.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { init } from 'ityped'
+
+const Description = () => {
+  const userName = useSelector(store => store.labyrinth.userName)
+  const actions = useSelector(store => store.labyrinth.actions)
+
+  useEffect(() => {
+    const textAnimation = document.querySelector("#textAnimation");
+    init(textAnimation, {
+      disableBackTyping: true,
+      showCursor: false,
+      strings: [actions.description]
+    })
+  }, [actions])
+
+  return (
+    <>
+      {userName !== null && <div className="history" id="textAnimation"></div>}
+    </>
```

Comment:
Cool to see the use of HTML DOM manipulation as the main point of a component instead of its return value :D